spread.yaml

```diff
@@ -64,8 +64,10 @@ backends:
                 workers: 6
 
             - debian-9-64:
-                image: debian-9-stretch
```

Comment:
So we're no longer testing stretch and instead test sid? Is that the thing that we want?

Reply:
AFAIU we're just using a different image, but something that's derived from `debian-9-64` automatically instead of `debian-9-stretch`

Reply:
In this case I just removed the image field because it is automatically getting the last debian-9 image. I mean, image: debian-9-stretch was not needed, but test does not change at all.

Reply:
Ah, this does make sense. Thank you for explaining this.